.eslintrc

```diff
@@ -14,6 +14,7 @@
   "rules": {
     "arrow-parens": ["error", "always"],
     "import/no-unresolved": "off",
+    "no-underscore-dangle": "off",
```

Comment:
Yeah, I think so. The codebase is heavily invested in underscore dangles. This rule allows us to do dependency injection easier. Example:

    const _config = sinon.stub();
    someFunction({ _config });